ioctl/config/configsetget.go

```diff
@@ -28,12 +28,14 @@ const (
 	_endpointPattern         = "(" + _ipPattern + "|(" + _domainPattern + ")" + "|(" + _localPattern + "))" + `(:\d{1,5})?`
 	_defaultAnalyserEndpoint = "https://iotex-analyser-api-mainnet.chainanalytics.org"
 	_defaultWsEndpoint       = "sprout-staging.w3bstream.com:9000"
+	// _defaultProjectRegisterContract  default project register contract address
+	_defaultProjectRegisterContract = "0x4F7e678B0203e0444E17512108dba4B08B39512e"
```

Comment:
register a new contract, for the "0x4F7e678B0203e0444E17512108dba4B08B39512e" contains much localhost uri